docs/basics/variable-info.md

```diff
@@ -0,0 +1,120 @@
+# Variable Information 
+
+This is page is to explain each variable inside .env file. We're break it down in three parts.
+
+### - Global Settings
+### - Backup Settings
+### - VPN Settings
+
+---
+
+## Global Settings
+
+<b>Global settings</b>: are the settings that apply to the local server and set initial default values for all virtual hosts running on this server.
+
+<b>COMPOSE_HTTP_TIMEOUT=60</b>: Configures the time (in seconds) a request to the Docker daemon is allowed to hang before Compose considers it failed. Defaults to 60 seconds.
+
+<b>DOCKERCONFDIR</b>: Your default application config files are stored. 
+
+<b>DOCKERGID</b> Docker group name
```

Comment:
Some variables have `:` and some don't?

Also we should be using markdown, not html. The build system will fail until this it's resolved.